.gitignore

```diff
@@ -0,0 +1,6 @@
+﻿################################################################################
+# This .gitignore file was automatically created by Microsoft(R) Visual Studio.
+################################################################################
+
+/xOfficeOnlineServer/xOfficeOnlineServer.sln

```

Comment:
Ping me offline to have a chat about using Visual Studio here - we are looking at moving away from it for xSharePoint - see [this discussion](https://github.com/PowerShell/xSharePoint/issues/172) for the highlights